### DISCUSSION TITLE
Iterate through locales in Blade

### DISCUSSION THREAD

**Discussion by ya-cha** [statamic/cms#7734] [Troubleshooting] [answered]
I want to access the Tag "locales" within a Blade component. As far as I understand the documentation it should work like this:

```php
Statamic::tag('locales')->fetch()
```

But this just returns an empty string. Since I am new to Statamic I would guess it's me ...

**Comment by joshuablum** [comment, marked as answer]
Hey @ya-cha,

This has been answered in this issue https://github.com/statamic/cms/issues/7502#issuecomment-1424847938.

You need to pass an `id` to the tag like this `@foreach (Statamic::tag('locales')->id($page->id) as $locale)`

Hope that helps :)

**Reply by ya-cha** [reply]
Thanks a lot, did not find this answer
